swap_meet/vendor.py

```diff
@@ -1,2 +1,76 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        inventory = inventory if inventory is not None else []
+        self.inventory = inventory
+
+    def add(self, new_item):
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self, remove_item):
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return remove_item
+        else:
+            return False
```

Comment:
Prefer using a guard clause first and then putting the main part of your logic after it not in an else block. This lets us emphasize the main part of the code by not indenting it.

```python
 if not item in self.inventory:
    return False

self.inventory.remove(item) 
return item
```